source/expressions.tex

```diff
@@ -3363,7 +3612,9 @@
 \indextext{initialization!parameter}%
 When a function is called, each parameter\iref{dcl.fct} is
 initialized\iref{dcl.init,class.copy.ctor} with
-its corresponding argument.
+its corresponding argument,
+and each precondition assertion of the function
+is evaluated.  %JMB: should we have an xref to dcl.contract.func here?
```

Comment:
Yes, please.  This is analogous to the xref to dcl.fct earlier in the sentence.

Reply:
done